ui/src/components/DataTypeField.vue

```diff
@@ -81,45 +85,17 @@ export default class extends Vue {
     return Object.values(paramValues).some((v) => !!v)
   }
 
-  onSelect (newValue: DataTypeOption) {
-    const newValueURI = newValue ? expand(newValue.uri) : null
+  onInput (newValue: DataTypeOption): void {
+    const newValueURI = newValue?.uri ?? null
```

Comment:
Conditional operator build fine? I'd thought it would need to adjust the build config

Reply:
Seems to be supported by TypeScript out of the box